app/models/course.rb

```diff
@@ -6,6 +6,7 @@ class Course < ApplicationRecord
   has_many :course_groups, dependent: :destroy
   has_many :instructors, through: :course_instructors
   has_many :course_queue_entries, through: :course_queues
+  enum sorting: [:timestamp, :number_of_resolved_requests_this_session]
```

Comment:
Thoughts on adding a [`_prefix`](https://api.rubyonrails.org/v5.0.7/classes/ActiveRecord/Enum.html) here so that something like `course.number_of_resolved_requests_this_session?` instead reads `course.sort_by_number_of_resolved_requests_this_session?`